pkg/reconciler/revision/resources/deploy_test.go

```diff
@@ -389,6 +408,42 @@ func withOwnerReference(name string) revisionOption {
 	}
 }
 
+func withPodSpec(containers []corev1.Container) revisionOption {
```

Comment:
Since this doesn't really take any podspec properties maybe we should call this `withContainers`